stingray/io.py

```diff
@@ -802,17 +802,22 @@ def _isattribute(data):
     ----------
     data : object
         The object to be checked.
+        The object must be list, list of tuples, tuple, tuple of tuples, tuple of lists and ndarrays.
 
     Returns:
         bool
         True if the data is a single number, False if it is an iterable.
     """
```

Comment:
It would help to add an `Examples` section, where a few doctests are implemented. E.g.
```
Examples
----------
>>> _isattribute(1)
True
>>> isattribute([2, 3, 5])
False
>>> isattribute([2])
False
```
etc.